feeds/combiner_test.go

```diff
@@ -51,14 +51,14 @@ func Test_combinerFeed_StartRange(t *testing.T) {
 				defer cancel()
 				cf, err := NewCombinerFeed(
 					ctx, CombinerFeedConfig{
-						APIUrl: "https://api.forta.network/graphql",
+						APIUrl: "https://api-dev.forta.network/graphql",
 					},
 				)
 				r.NoError(err)
 				cf.AddSubscription(
 					&protocol.CombinerBotSubscription{
-						BotId:   "0x5e13c2f3a97c292695b598090056ba5d52f9dcc7790bcdaa8b6cd87c1a1ebc0f",
-						AlertId: "FORTA-6",
+						BotId:   "0x2bee737433c0c8cdbd924bbb68306cfd8abcf0e46a6ce8994fa7d474361bb186",
+						// AlertId: "FORTA-6",
```

Comment:
should we uncomment this? also, is alertId required for the subscriptions i.e. if I wanted to get all alerts from a bot, do I need to know all of its possible alert IDs?

Reply:
This PR prevents that behavior:
https://github.com/forta-network/forta-core-go/pull/147